src/include/daos_fs.h

```diff
@@ -253,6 +253,36 @@ int
 dfs_readdir(dfs_t *dfs, dfs_obj_t *obj, daos_anchor_t *anchor,
 	    uint32_t *nr, struct dirent *dirs);
 
+/**
+ * User callback defined for dfs_readdir_size.
+ */
+typedef int (*dfs_readdir_cb_t)(dfs_t *dfs, dfs_obj_t *obj, const char name[],
+				void *_udata);
+
```

Comment:
We usually use the term filler() for this kind of callback function.

Reply:
sure, i can rename it